ui/packages/catalog-ui-search/src/main/webapp/react-component/container/multi-select-actions/index.tsx

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
❓ Is there a reason this class is the same as https://github.com/codice/ddf/pull/4595/files#diff-f45927c091638e6fd42db33875f8bd08R12?

Is it because as we add functionality in the future, these two classes will start to look different?

Reply:
In general, we're having each React component have a container part, which deals with business logic, and a presentational part, which only cares about how the data looks. The container component will pass props to the presentational component. https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0

I was thinking that it might be good to keep the same concept, but instead group all the files related to a component in the same place.